core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -366,8 +362,8 @@ trait WhiskActionsApi
         }
 
         WhiskAction(
-            namespace,
-            name,
+            entityName.path,
```

Comment:
this will require a more invasive change which is tied to removing the WhiskEntity layer.